app/src/main/res/values/dimens.xml

```diff
@@ -6,7 +6,16 @@
     <dimen name="space_medium">20dp</dimen>
     <dimen name="space_large">32dp</dimen>
     <dimen name="space_extra_large">64dp</dimen>
+
     <!--  item_user  -->
     <dimen name="profile_img_size">100dp</dimen>
     <dimen name="placeholder_corner_radius">50dp</dimen>
+    <dimen name="size_24">24dp</dimen>
```

Comment:
Not a good naming choice.

Reply:
Appropriate naming given as 'card_corner_radius'.